src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -75,7 +74,7 @@ object Run  {
 
       // Update the database state:
       val newBackendInfo = JesCallBackendInfo(db.CallStatus(ExecutionStatus.Running.toString), Option(JesId(run.jesId)), Option(JesStatus(currentStatus.toString)))
-      dataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
+      DataAccess.instance.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
```

Comment:
You could do `import DataAccess._` and then use `instance` directly ? If you call it `dataAccess` instead of `instance` it's almost seamless compared to how it was before.